litclock_annotated.csv

```diff
@@ -187,23 +202,25 @@
 01:38|one-thirty-eight|At one-thirty-eight am suspect left the Drive-In and drove to seven hundred and twenty three North Walnut, to the rear of the residence, and parked the car.|The Narc|William Edmund Butterworth|unknown
 01:40|one-forty am|March twelfth, one-forty am, she leaves a group of drinking buddies to catch a bus home. She never makes it.|Bones to Ashes|Kathy Reichs|unknown
 01:44|sixteen minutes to two|She knew it was the stress, two long days of stress, and she looked at her watch, sixteen minutes to two, and she almost leaped with fright, a shock wave rippling through her body, where had the time gone?|Trackers|Deon Meyer|unknown
+01:45|1:45|In the end I accepted a glass of claret. It seemed both churlish and censorious to let him drink alone. The carriage-clock stuck 1:45 before he roused himself and said: "Sorry, Cousin. Rather drunk. Be glad of your shoulder. To bed, to sleep, perchance to dream."|The Mistletoe Murder|P.D. James|sfw
```

Comment:
Realizing that I marked this as sfw, but worth bringing up for discussion whether quotes with alcohol should be marked as nsfw? Obviously not a trigger for me at all, but there's a variety of personal or cultural reasons someone might not be comfortable with alcohol references. Could see it going either way. Or possibly figuring out more granular content controls than just sfw/nsfw at some point in the future.

Reply:
I think it is good to leave it as sfw (at least for now). But I agree that it is controversial. Generally I think the nsfw tag should be used quite liberally to make to filter out quotes you might not want showing on your work screensaver or in e.g. a public library.